src/components/common/StackedTextBotBox.tsx

```diff
@@ -0,0 +1,70 @@
+import AcUnitIcon from '@mui/icons-material/AcUnit';
+import CheckIcon from '@mui/icons-material/Check';
+import { Box, BoxProps, Typography, useTheme } from '@mui/material';
+import React from 'react';
+
+export interface StackedTextBotBoxProps extends BoxProps {
+  type?: 'normal' | 'large' | undefined;
+  titleColor?: string | undefined;
+  status?: 'Active' | 'On Ice' | 'Frozen' | undefined;
+}
+
+export const StackedTextBotBox: React.FC<StackedTextBotBoxProps> = ({
```

Comment:
was the creation of this file an accident?

looks like we have a `StackedText`, `StackedTextBox` and a `StackedTextHLBox`. At the very least, can a clearer name be chosen and have this new component be an extension of one of the others?